### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by rrrutledge**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+* When submitting new files, follow the folder structure listed under **Repository Setup** in [the README](./README.md).
+* Write each new sentence on a new line.
+GitHub allows leaving comments on a line-by-line basis.
+Review and comment on the content of submitted text is much easier if there are multiple lines on-which to leave comments.
+Sentences on consecutive lines will be collapsed into a single paragraph (like this one) in the  final rendring of the content.
+* When reviewing a pull request, make suggestions via regular comment.
+If you have a specific text to suggest, use the [the _GitHub_ "suggestion" feature][suggestion feature].
+
+
+[suggestion feature]: https://help.github.com/articles/commenting-on-a-pull-request/#adding-line-comments-to-a-pull-request
```
Comment (lenucksi) [inline CONTRIBUTING.md]:
```suggestion
[suggestion feature]: https://help.github.com/articles/commenting-on-a-pull-request/#adding-line-comments-to-a-pull-request and https://help.github.com/articles/incorporating-feedback-in-your-pull-request/
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Thanks for contributing.  A few tips:
+
+* When submitting new files, follow the folder structure listed under **Repository Setup** in [the README](./README.md).
+* Write each new sentence on a new line.
+GitHub allows leaving comments on a line-by-line basis.
+Review and comment on the content of submitted text is much easier if there are multiple lines on-which to leave comments.
+Sentences on consecutive lines will be collapsed into a single paragraph (like this one) in the  final rendring of the content.
```
Comment (lenucksi) [inline CONTRIBUTING.md]: Nice paragraph. 
```suggestion
Sentences on consecutive lines will be collapsed into a single paragraph (like this one) in the final rendering of the content.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Thanks for contributing.  A few tips:
+
+* When submitting new files, follow the folder structure listed under **Repository Setup** in [the README](./README.md).
+* Write each new sentence on a new line.
```
Comment (lenucksi) [inline CONTRIBUTING.md]:
```suggestion
* Write each new sentence on a new line.
Please don't insert non-sentence or structure-intended extra line breaks. That allows for easier editing and re-arrangement of text parts.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (rrrutledge) [reply CONTRIBUTING.md]: This line is not visible in the rendered markdown.  It creates a shorthand way of representing the help article URL using the friendly text `suggestion feature` instead.  You can see me taking advantage of that in line 9.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rrrutledge) [reply CONTRIBUTING.md]: Thanks for this fix.

File: CONTRIBUTING.md (hunk 3 above)
Comment (rrrutledge) [reply CONTRIBUTING.md]: This suggested text has multiple sentences on a single line :).  I'll take a stab at re-writing.  Also will phrase instructions in the positive tense.

Comment (rrrutledge) [reply CONTRIBUTING.md]: Take a look at the latest.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rrrutledge) [reply CONTRIBUTING.md]: Included your link inline in the contributing text.

Comment (lenucksi) [reply CONTRIBUTING.md]: Nice, that's a feature I didn't know yet. 👍

Comment (lenucksi) [reply CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 3 above)
Comment (lenucksi) [reply CONTRIBUTING.md]: > This suggested text has multiple sentences on a single line :). 
That's indeed true, ideal place to actually put multiple sentences on a single line! ;) Thanks for catching it!

> I'll take a stab at re-writing. Also will phrase instructions in the positive tense.
Looks good to me!

**Comment by lenucksi** [comment]
This looks good to me now with @rrrutledge's changes. 👍 
Let's merge it in!

**Pull request merged**